src/main.rs

```diff
@@ -401,36 +401,31 @@ fn run() -> Result<(), failure::Error> {
     if let Some(matches) = matches.subcommand_matches("config") {
         let api_key = matches.is_present("api-key");
 
-        let mut user = GlobalUser {
-            email: None,
-            api_key: None,
-            api_token: None,
-        };
+        let mut user: GlobalUser;
 
         if !api_key {
```

Comment:
nit: if not is weird to read, i know it's the "preferred" way but i think flipping the if/else cases is more readable, and also follows the pattern of handling "special" or "weird" or "exit" cases first; "early return".

Reply:
how about i actually get the opposite of `api_key` when I pull it from the match, so this if-else doesn't have a `!`?